cl/favorites/templates/user_prayers.html

```diff
@@ -121,6 +121,85 @@ <h1 class="text-center v-offset-below-3">{% if is_page_owner %}Your PACER Docume
+              <th>Case Name</th>
+              <th>Document Number</th>
+              <th>Document Description</th>
+              <th>Requested On</th>
+              <th>Status</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for rd in rd_with_prayers_granted %}
+            <tr>
+              <td>{{ rd.docket_entry.docket.court.citation_string }}</td>
+              {% with docket=rd.docket_entry.docket %}
+              <td>
+                <a href="{% url "view_docket" docket.pk docket.slug %}">
+                  {{ docket|best_case_name|safe|v_wrapper }} ({{ docket.docket_number }})
+                </a>
+              </td>
+              <td>
+                <a href="{% url "view_recap_document" docket.pk rd.docket_entry.entry_number docket.slug %}">
+                  {{ rd.document_number }}
+                </a>
+              </td>
+              {% endwith %}
+              <td>{{ rd.description }}</td>
+              <td>{{ rd.prayer_date_created|date:"M j, Y"  }}</td>
+              <td>{% if rd.prayer_status == 1 %} Pending {% elif rd.prayer_status == 2 %}
+                <div class="btn-group hidden-xs col-sm-4 col-md-3 hidden-print flex">
+                  {% if rd.filepath_local %}
+                    <a href="{{ rd.filepath_local.url }}"
+                        rel="nofollow"
+                        role="button"
+                        class="btn btn-primary btn-xs"
+                            {% if rd.date_upload %}
+                        title="Uploaded {{ rd.date_upload|datetime_in_utc }}"
+                            {% endif %}>
+                      Download PDF
+                    </a>
+                    <button type="button"
+                            class="btn btn-primary btn-xs dropdown-toggle"
+                            data-toggle="dropdown"
+                            aria-haspopup="true"
+                            aria-expanded="false">
+                      <span class="caret"></span>
+                      <span class="sr-only">Toggle Dropdown</span>
+                    </button>
+                    <ul class="dropdown-menu">
+                      <li>
+                        <a href="{{ rd.filepath_local.url }}" rel="nofollow">From
+                          CourtListener</a>
+                      </li>
+                      {% if rd.filepath_ia %}
+                        <li>
+                          <a href="{{ rd.filepath_ia }}"
+                              rel="nofollow">From
+                            Internet Archive</a>
+                        </li>
+                      {% endif %}
+                    </ul>
+                    {% endif %}
+                </div>
+                {% endif %}
+              </td>
+            </tr>
+            {% empty %}
+            <tr>
+                <td colspan="2">No document requests granted. Consider making one!</td>
+            </tr>
+            {% endfor %}
```

Comment:
Let's hide this table until we have some granted requests to showcase.  It's not very useful without them.

<img width="1169" alt="Screenshot 2025-02-18 at 10 31 00 AM" src="https://github.com/user-attachments/assets/bb29aa69-9d61-44b5-b201-d752d3f5d89c" />